website/src/docs/changelog.md

```diff
@@ -852,7 +851,511 @@ it a go and let us know what you think via a
   with the `defer:` keyword
   ([Documentation](https://taskfile.dev/#/usage?id=doing-task-cleanup-with-defer),
   #475, #626).
-- Remove long deprecated and undocumented `$` variable prefix and `^` command
+- Remove long deprecated and undocumented `---
+  title: Changelog
+  description: Historique complet des versions et changements de Task, de la v3.44.1 jusqu'à la v1.0.0
+  permalink: /changelog/
+  outline: deep
+---
+
+# Changelog
```

Comment:
Looks like a copy paste error?

Reply:
Sorry, comment wasn't very clear. The file starts at v3.44.1 and goes all the way back to v3.10.0, then on line 861, it looks like a copy of the file has been inserted again all the way until line 1364-ish. It's much clearer if you look at the full diff on GitHub.

Reply:
Fixed !